lib/screens/home/repository/wod_repository.dart

```diff
@@ -13,4 +15,8 @@ class WodRepository {
           .toList();
     });
   }
+
+  Future<void> addWod(data) {
+    return _firestore.collection('wods').add(data);
```

Comment:
Just a tip: sometimes it makes sense to make some data validation before inserting it into the database. For instance, we want to make sure that the date that was inserted by the user is from today or the future, right? It doesn't make sense to insert a Wod for yesterday. Or does it?

This is just an example but this is some kind of validation that might make sense.
You can also check it in the form but I would prefer to do it before saving it on the database because, if you want to add WoDs in other part of the app, you wouldn't need to to the validation the that new form again because it would always be inside `addWod` method.